botocore/exceptions.py

```diff
@@ -823,3 +823,12 @@ class InvalidChecksumConfigError(BotoCoreError):
         'Unsupported configuration value for {config_key}. '
         'Expected one of {valid_options} but got {config_value}.'
     )
+
+
+class NoSupportedProtocolError(BotoCoreError):
```

Comment:
What do you think of a name like `UnsupportedServiceProtocolError` instead?

Reply:
`UnsupportedServiceProtocolsError` is another alternative, I just want to make sure that protocols is plural